RELEASING/email_templates/announce.tmpl

```diff
@@ -0,0 +1,39 @@
+To: $receiver_email
```

Comment:
Oh, interesting to see `string.Template` here. I'd have gone Jinja, but that's probably overkill, whatever works.

Reply:
trying to keep it away from any requirements, let's see if I can pull it off. Already feeling the need for a loop block on the template to render a list of binding votes

Reply:
ended up using Jinja2 and Click much cleaner and maintainable code now